src/tinypeel/Peeling/PeelingInfo.py

```diff
@@ -102,8 +109,8 @@ def setupTransmission(length, peelingInfo):
     if peelingInfo.positions is None:
         localMap = np.linspace(0, 1, num=peelingInfo.nLoci, dtype=np.float32)
     else:
-        localMap = (
-            peelingInfo.positions / peelingInfo.positions[-1]
+        localMap = (peelingInfo.positions - peelingInfo.positions[0]) / (
+            peelingInfo.positions[-1] - peelingInfo.positions[0]
         )  # This should be sorted. Need to add in code to check.
```

Comment:
@XingerTang do we know if `InputOutput.readMapFile()` checks for sorted output or sorts the input? Please open an issue.

Reply:
@gregorgorjanc `InputOutput.readMapFile()` does not check, we could implement it but it would be quite expensive to actually write a sorting algorithm (a quick research gives the minimum would be O(nlog(n))).

Reply:
@XingerTang also then we should add a test where you scramble the order of rows in the map file and correspondingly the columns of the genotype file. The question now is what should be the output - in the sorted order or the original order (another option, doh)?

But, should some of this functionality be in `tinyhouse` or in `AlphaPeel`?

Best to move this discussion into an issue!

Reply:
@gregorgorjanc All of these functionalities are in `tinyhouse`.

Reply:
> @gregorgorjanc `InputOutput.readMapFile()` does not check, we could implement it but it would be quite expensive to actually write a sorting algorithm (a quick research gives the minimum would be O(nlog(n))).

I know! Well, we have the issue now open at https://github.com/AlphaGenes/AlphaPeel/issues/155 so I propose we remove the comment in row 114 since this test/check should be done elsewhere and that comment will be redundant and in the wrong place anyway.

Reply:
> @gregorgorjanc The issue is opened. But I think we don't have to be too worried about the genotype input, it has an option for the output order,`out_id_order`, that allows the output to be in the order of ids or the sequence of the input of different files.

That is for the order of rows. Map file positions deal with columns. Anyway, let's drop this discussion for later in https://github.com/AlphaGenes/AlphaPeel/issues/155

Reply:
> > @gregorgorjanc `InputOutput.readMapFile()` does not check, we could implement it but it would be quite expensive to actually write a sorting algorithm (a quick research gives the minimum would be O(nlog(n))).
> 
> I know! Well, we have the issue now open at #155 so I propose we remove the comment in row 114 since this test/check should be done elsewhere and that comment will be redundant and in the wrong place anyway.

Sure!